kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2667,6 +2667,20 @@ object KyuubiConf {
       .stringConf
       .createOptional
 
+  val ENGINE_CHAT_PROXY_HTTP_URL: OptionalConfigEntry[String] =
+    buildConf("kyuubi.engine.chat.proxy.url")
+      .doc("Whether to use a proxy to access Chat API. e.g. http://127.0.0.1:1087")
+      .version("1.8.0")
+      .stringConf
+      .createOptional
+
+  val ENGINE_CHAT_HTTP_TIMEOUT: ConfigEntry[Long] =
+    buildConf("kyuubi.engine.chat.http.timeout")
+      .doc("Timeout for calling Chat API")
```

Comment:
```suggestion
      .doc("Timeout in seconds of API calling in Chat engine")
```

Reply:
maybe we can respect the http client parameter here